tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md

```diff
@@ -0,0 +1,100 @@
+---
+title: TiDB Cloud TPC-C Performance Test Report
+summary: Introduce the TPC-C performance test results for TiDB Cloud.
+---
+
+# TiDB Cloud TPC-C Performance Test Report
+
+This document provides the TPC-C performance test result for the TiDB Dedicated cluster with a cluster version of v7.1.0. This report can be used as a reference for the performance of TiDB Self-Hosted v7.1.0 clusters.
```

Comment:
```suggestion
This document provides the TPC-C performance test steps and results for a TiDB Dedicated cluster with the TiDB version of v7.1.0. This report can also be used as a reference for the performance of TiDB Self-Hosted v7.1.0 clusters.
```

Reply:
TiDB version is used on UI
![image](https://github.com/pingcap/docs/assets/79440533/379c12a5-9aad-4783-90ed-dd2fa9a5b38b)